app/javascript/packages/document-capture/components/acuant-camera.tsx

```diff
@@ -210,9 +213,13 @@ export interface AcuantSuccessResponse {
    */
   image: AcuantImage;
   /**
-   * Document type
+   * Document type for Acuant SDK 11.8.2
    */
   cardtype: AcuantDocumentType;
+  /**
+   * Document type for Acuant SDK 11.9.1
+   */
+  cardType: AcuantDocumentType;
```

Comment:
I think this typing is a little off, since it is saying that both `cardtype` and `cardType` will be present. You could represent the 'legacy' version by removing `cardtype` from this interface and adding a separate type like this:

```typescript
export type LegacyAcuantSuccessResponse = Omit<AcuantSuccessResponse, "cardType"> & {
  cardtype: AcuantDocumentType
};
```

Then you can change references to `AcuantSuccessResponse` to `AcuantSuccessResponse | LegacyAcuantSuccessResponse` and disambiguate by looking for the presence of `cardtype` in the response:

```
if ('cardtype' in response) { 
  // it's a legacy!
} else {
  // it's not a legacy
}
```

Reply:
I'll try this, thanks for suggesting it.  I had initially tried `cardtype?: AcuantDocumentType` and `cardType?: AcuantDocumentType`.  Since that translates to `cardType: AcuantDocumentType | undefined`, it broke `AcuantDocumentType[documentType]` in `getDocumentTypeLabel`.